bitbots_blackboard/bitbots_blackboard/capsules/team_data_capsule.py

```diff
@@ -62,7 +61,7 @@ def __init__(self, node: Node):
             'body.localization_precision_threshold.theta_sdev').value
 
     def is_valid(self, data: TeamData) -> bool:
-        return self.node.get_clock().now() - Time.from_msg(data.header.stamp) < Duration(seconds=self.data_timeout) \
+        return self.node.get_clock().now() - RclpyTime.from_msg(data.header.stamp) < Duration(seconds=self.data_timeout) \
```

Comment:
Why change it from Time to RclpyTime?

Reply:
`RclpyTime` is just an alias for `Time` as you can see in line 10.
I have done this to more easily distinguish between the two very different classes called `Time` in ROS 2:

- `from rclpy.time import Time as RclpyTime`
- `from builtin_interfaces.msg import Time as MsgTime`

We have confused both of these several times (unintended pun), which lead to crashes.